src/leaguepedia_client.py

```
@@ -112,12 +142,174 @@ async def get_scoreboard_data(self, overview_page: str):
+                    if bot:
+                        try:
+                            msg = (
+                                f"Leaguepedia rate limit hit for "
+                                f"{overview_page}. Backing off for "
+                                f"{cooldown_minutes:.1f} minutes."
+                            )
+                            asyncio.create_task(
+                                send_admin_update(
+                                    msg, mention_user_id=mention_id
+                                )
+                            )
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)